- Gene Expression Control
	- All somatic cells are genetically **identical**
	- Gene expression is mainly controlled through *transcriptional regulation* 
	- Cells differ through specialization by activating specific genes 
- Regulation of Gene Expression in Prokaryotes
	- Reflects life history, as it has a rapid and reversible response to the environment
	- *Operon*: A cluster of prokaryotic genes and DNA regulatory sequences
	- RNA polymerase binds at the *promoter* for the operon
	- Many genes may be transcribed into one mRNA
	- Cluster of genes is called a *transcription unit*
- **Operon**
	- *Operator*
		- Regulatory DNA sequences in operon 
		- Regulatory proteins bind the operator
		- **Repressor proteins**
			- Prevent operon from being expressed
		- **Activator proteins**
			- Turn on expression of genes from operon
	- *lac* operon for Lactose Metabolism
		- lactose metabolism In E Coli requires 3 genes:
			- lacZ, lacY and lacA
			- The *lac operon* contains all three genes and regulatory sequences
			- The lac operon operator sequence is between the promoter and lacZ
	- When lactose is absent?
		- *Lac repressor*
			- Active lac repressor expressed from the lacI gene binds to operator
			- RNA polymerase cannot bind to promoter
			- **Transcription does not occur**
	- When lactose is present?
		- *Allolactose* -> made from lactose
			- Inducer of lac operon by binding to lac repressor, inactivates it
			- RNA polymerase binds to the promoter
			- **Transcription occurs**
	- Positive Regulation of lac Operon
		- lac Operon only operates when lactose but not glucose is present, this is due to glucose being a more efficient energy source than lactose
		- Means the operon is simply not needed
		- Catabolite Activator Protein (CAP)
			- Activator that stimulates gene expression
			- CAP activated by cAMP
			- **cAMP only abundant when glucose levels are low**
		- This means that when glucose is present and lactose is present, transcription levels are low because CAP not activated leading to RNA polymerase not binding
	- trp Operon Genes
		- 5 genes: trpA, trpB, trpC, trpD, trpE
		- All of these needed for synthesis of tryptophan by trp operon
			- Trp repressor encoded by trpR elsewhere in genome
			- Trp repressor synthesized in inactive form 
	- trp Operon Regulation
		- Default state of trp operon is expression since Trp repressor is **inactive**
			- It is *activated* by the presence of **tryptophan**
		- trp operon is **repressible operon** (usually on)
			- Tryptophan is corepressor (combines with repressor to shut off)
		- lac and trp operons both exhibit negative gene regulation
	- Tryptophan is Absent?
		- Means it needs to be made **in the cell**
		- trp repressor is inactive and cannot bind to the operator
		- RNA polymerase binds and transcription happens
		- Ribosomes recognize the ribosome binding sites
		- **OPERON ON**
	- Tryptophan is Present?
		- Enters the cell and acts as a corepressor by binding to the inactive trp repressor and activating it
		- RNA polymerase unable to bind, no transcription
		- **OPERON OFF**
	- Regulation of Gene Expression in Eukaryotes
		- 4 levels:
			- Transcriptional Regulation (*Most important*)
				- Chromatin remodelling to make genes accessible for transcription
				- Regulatory events at a genes promoter and regulatory sequences
			- Posttranscriptional regulation
				- Variations in pre-mRNA processing
				- Removal of masking proteins
				- Variations in rate of mRNA breakdown
				- RNA interference
			- Translational Regulation
				- Variations in rate of initiation of protein synthesis
			- Posttranslational regulation
				- Variations in rate of protein processing
				- Removal of masking segments
				- Variations in rate of protein breakdown
	- Transcriptional Regulation
		- *Transcription Initiation*
			- Promoter 
				- Includes TATA box that binds transcription factors
			- Promoter proximal region
				- Contains promoter proximal elements
				- Stimulates or Inhibits transcription rates
			- Enhancer further upstream
				- Determines max/overall transcription rate
			- General transcription factors initiate transcription
				- Bind to TATA box area
				- Recruit RNA polymerase II
				- Orientate enzyme to start transcription at the correct place 
			- **Activators** bind to promoter proximal elements and increase transcription rate
			- Activators are proteins with 2 distinct *domains*
			- 3D arrangement of proteins produces highly specialized regions called *motifs*
				- Helix-turn-helix
				- Zinc Finger
				- Leucine Zipper
			- **Coactivators** bridge enhancer and promoter 
				- Increase transcription to maximal rate
			- **Repressors** oppose effect of activators 
				- Decrease transcription 
			- Combinatorial Gene Regulation
				- **Complex** gene regulation with **limited/small** numbers of activators.
				- Allows genes with related functions to respond to 1 signal
			- Hormones
				- Signal molecules activate all cells with specific hormone receptors
				- All genes regulated by a specific hormone contain a *steroid hormone response element*
		- *Methylation of DNA*
			- DNA methylation
				- Adds methyl groups to cytosine
				- *Gene Silencing* occurs when DNA methylation is located in promoters
			- Genomic Imprinting
				- Permanent silencing of a maternal/paternal allele
				- Inherited methylated allele is inactive
				- Methylation maintained as DNA is replicated
		- *Chromatin Structure*
			- Eukaryotic DNA +Histones = nucleosomes
				- Promoters located in nucleosome
			- Chromatin Remodelling
				- Activators recruit *remodelling complexes* taht displace 